docs/indicator-configuration.md

```diff
@@ -0,0 +1,470 @@
+
+## Note about unit-specific and series-specific settings
+
+Several indicator configuration settings can be limited to a particular Unit and/or Series. For example, the `graph_titles` setting can be configured like this:
+
+```
+graph_titles:
+  - unit: Percent
+    title: My title for percent
+  - unit: Total
+    title: My title for total
+```
+
+Indicator setting for a specific Series-Unit combination can be configured like this:
+
+```
+graph_titles:
+  - series: Wastewater treated
+    unit: Percent
+    title: My title for percent of wastewater treated
+```
+
+In addition to graph_titles, the other fields like this include:
+
+* graph_annotations
+* graph_limits
+* precision
+
+These fields are described below. Note that if a unit/series is not specified, then the item will apply to any unit/series. For example:
+
+```
+graph_titles:
+  - series: SERIES123
+    title: This title will appear for SERIES123 only
+  - series: ''
+    title: This title will appear for all other series
+```
+
+## Indicator configuration fields
+
+### composite_breakdown_label
+
+**_Optional_** This setting is used when importing data from SDMX to specify a more useful label for the COMPOSITE_BREAKDOWN column (if present). Translation keys are supported, as always. Using the example below would change the COMPOSITE_BREAKDOWN label to "Hazard type" for this indicator:
+
+```nohighlight
+composite_breakdown_label: Hazard type
+```
+
+### computation_units
+
+**_Recommended_**: This setting is used to specify the unit used in the headline data for the indicator.
+
+```nohighlight
+computation_units: Metric tons
+```
+
+With translation key:
+
+```nohighlight
+computation_units: my_translations.metric_tons
+```
+
+### copyright
+
+**_Optional_**: This setting can be used to display copyright information below the graph and table on the indicator page.
+
+```nohighlight
+copyright: Copyright 2019 - My organisation
+```
+
+With translation key:
+
+```nohighlight
+copyright: my_translations.copyright_message
+```
+
+### data_footnote
+
+**_Optional_**: This setting can be used to display additional inforamtion about the data below the graph and table on the indicator page.
+
+
+```nohighlight
+data_footnote: My additional information
+```
+
+With translation key:
+
+```nohighlight
+data_footnote: my_translations.1-1-1-footnote
+```
+
+
+### data_not_statistical
+
+**_Required_**: This setting is used to specify whether the indicator is statistical (can be charted/graphed) or not. If you have uploaded data that you want to display on a chart/table for an indicator, set this to `false`.
+
+```nohighlight
+data_non_statistical: false
+```
+
+You may want to display some very important information which site viewers must keep in mind when using the data provided. To display a notice above the graph in a coloured box, you can use the following fields within the metadata file.
```

Comment:
I think this needs to have a heading above it, like maybe: `### Data notice settings`. Or optionally it could be moved below `### data_notice_class` since that is the first one.